### PR TITLE
Chore(Exp-docs): Add Locust Loadgen Chaos Experiment Docs

### PR DESCRIPTION
**Pull request by uditgaurav**
Locust loadgen chaos fault documentation

### CODE REVIEW

**Review by avaakash — CHANGES_REQUESTED** [review]

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen-chaos.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+id: locust-loadgen-chaos
+title: Locust Loadgen Chaos
+---
+Locust loadgen chaos fault causes load generation on the given target hosts for a specified chaos duration
+- It can result in the slowness or unavailability of the target host due to heavy load.
+- This fault checks the performance of the application (or process) running on the instance.
+
+![Locust Loadgen Chaos](./static/images/locust-loadgen-chaos.png)
```
[the remark this answers is not in the document]
Comment (uditgaurav) [reply docs/chaos-engineering/chaos-faults/load/locust-loadgen-chaos.md:9]: It is not a aws chaos

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    <h3>Optional fields</h3>
+    <table>
+        <tr>
+            <th> Variables </th>
+            <th> Description </th>
+            <th> Notes </th>
+        </tr>
+        <tr>
+            <td> TOTAL_CHAOS_DURATION </td>
+            <td> Duration that you specify, through which chaos is injected into the target resource (in seconds). </td>
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/chaos-faults/load/locust-loadgen.md:64]: Can this be rewritten to "Time taken to inject chaos into a target resource (in seconds)."

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+            <td> Defaults to 60s. For more information, go to <a href="https://developer.harness.io/docs/chaos-engineering/chaos-faults/common-tunables-for-all-faults/#chaos-interval"> chaos interval.</a></td>
+        </tr>
+        <tr>
+            <td> USERS </td>
+            <td> Peak number of concurrent Locust users causing the load. </td>
+            <td> Defaults <code>30</code>. For more information, go to <a href="https://developer.harness.io/docs/chaos-engineering/chaos-faults/load/locust-loadgen-chaos#number-of-users"> number of users.</a></td>
+        </tr>
+        <tr>
+            <td> SPAWN_RATE </td>
+            <td> Rate to spawn users at (users per second).</td>
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/chaos-faults/load/locust-loadgen.md:79]: "Numbers of users spawned per second."

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+            <td> Defaults <code>chaosnative/locust-loadgen:latest</code>. For more information, go to <a href="https://developer.harness.io/docs/chaos-engineering/chaos-faults/load/locust-loadgen-chaos#custom-load-image"> custom load image.</a></td>
+        </tr>
+        <tr>
+            <td> LOAD_TYPE </td>
+            <td> Used as a suffix in the load file name. </td>
+            <td> Defaults to <code>load</code>. </td>
+        </tr>
+        <tr>
+            <td> RAMP_TIME </td>
+            <td> Period to wait before and after injecting chaos (in seconds). </td>
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/chaos-faults/load/locust-loadgen.md:99]: Can we say "The wait period before and after injecting chaos (in seconds)."

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+            <td> For example, 30s. For more information, go to <a href="https://developer.harness.io/docs/chaos-engineering/chaos-faults/common-tunables-for-all-faults/#ramp-time"> ramp time.</a></td>
+        </tr>
+    </table>
+
+
+### Target host
+
+It specifies the value of the target host under chaos. Tune it by using the `HOST` environment variable.
+
+Use the following example to tune it:
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/chaos-faults/load/locust-loadgen.md:109]: Use the following example to tune the target host.

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+        env:
+        - name: HOST
+          value: 'https://www.google.com'
+```
+
+### Number of users
+
+It specifies the number of users or workers involved in the load generation. Tune it by using the `USERS` environment variable.
+
+Use the following example to tune it:
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/chaos-faults/load/locust-loadgen.md:134]: Use the following example to tune the number of users.

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+          value: '100'
+        - name: HOST
+          value: 'https://www.google.com'
+```
+
+### Custom load image
+
+It specifies the rate at which users are spawned (users spawned per second). Tune it by using the `LOAD_IMAGE` environment variable.
+
+Use the following example to tune it:
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/chaos-faults/load/locust-loadgen.md:161]: Use the following example to tune the custom load image.

File: docs/chaos-engineering/chaos-faults/load/locust-loadgen.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+        env:
+        - name: USERS
+          value: '100'
+        - name: HOST
+          value: 'https://www.google.com'
+```
+
+### Custom load image
+
+It specifies the rate at which users are spawned (users spawned per second). Tune it by using the `LOAD_IMAGE` environment variable.
```
Comment (roshnisarangadharan) [inline docs/chaos-engineering/chaos-faults/load/locust-loadgen.md:159]: ...users are spawned per second.

**Review by roshnisarangadharan — APPROVED** [review]
Reviewed the changed sections only. Approved with some suggestions. Implement or ignore as needed.

**Review by roshnisarangadharan — APPROVED** [review]

**Review by SudheendraKatte — DISMISSED** [review]
LGTM

**Review by roshnisarangadharan — APPROVED** [review]

**Pull request merged**